packages/nocodb-sdk/src/lib/sqlUi/MssqlUi.ts

```diff
@@ -306,7 +190,7 @@ export class MssqlUi {
         return '';
 
       case 'decimal':
-        return '';
+        return '10';
```

Comment:
We return number for nvarchar and varchar, if it is more appropriate to return string we might fix others

Reply:
@mertmit Actually, the type doesn't matter here except `0`(number) since its falsy value may get ignored. I will make it consistent anyway.